### ISSUE TITLE
Font family alias warnings on macOS

### ISSUE THREAD

**Issue by anjohnson** [eteran/nedit-ng#327]
I see this warning displayed on my Mac terminal windows where I started nedit-ng when it opens some dialogs or the "Output from Command" window:
```
Warning: Populating font family aliases took 145 ms. Replace uses of missing font family "Monospace"
with one that exists to avoid this cost. 
```
On my ancient laptop the time taken is somewhat longer (this was taken on a much faster iMac). Is there some easy way to configure Nedit to use a font that does exist on macOS instead of "Monospace"? It would be nice to silence the warning, although it doesn't happen that often so this is definitely a low priority.

Thanks!

**Comment by eteran** [comment]
Yeah, there should be some monospace font we can default to on mac that will be "always there". Using "Monospace", it was hoped that that was generic enough that it would always be there on every platform. But we'll need to work out a slightly better solution.

**Comment by anjohnson** [comment]
I've taken to manually substituting `Menlo` for `Monospace` in the `src/*.ui` files before building. I also see the font name used in the french translations, but I haven't bothered there for my builds. Is there some way to substitute a macro/variable in the ui files which could be set through a CMake option at build-time?

**Comment by eteran** [comment]
Oh that's good info, I'll see if there's an easy way to specify using that for macOS only or something like that. I'll add it to the TODO list!
